src/UIAutomation/Microsoft.UI.UIAutomation/AutomationRemoteOperation.cpp

```diff
@@ -152,6 +152,15 @@ namespace winrt::Microsoft::UI::UIAutomation::implementation
         return make<AutomationRemoteAnyObject>(newId, *this);
     }
 
+    winrt::AutomationRemoteAnyObject AutomationRemoteOperation::NewEmpty()

```

Comment:
Could you make a comment stating why this is different from our current method of passing operand IDs as output into instructions? Like, give a scenario where connection bound objects/pattern extension make this necessary?

Reply:
This is almost similar to our NewNull, except NewNull inserts an instruction. To avoid wasting instructions, NewEmpty is used. It just assigns an operand id and doesn't insert any instruction.

Reply:
OK, I thought about this some more. Is this for if extensions want one of their operands to be an out param? In that case, I'd use that as an example in the comment. Maybe say something like this allows you to specify an out parameter for `CallExtension` without wasting an instruction on creating a null object.